### PR TITLE
IDEX-2269: Create UI to set svn:ignore property (define files and directories for SVN to ignore)

### PR DESCRIPTION
**Pull request by vzhukovskii**
@stour @skabashnyuk @vparfonov

### CODE REVIEW

File: plugin-svn/che-plugin-svn-ext-subversion/src/main/java/org/eclipse/che/ide/ext/svn/client/SubversionClientServiceImpl.java [hunk 1]
```diff
@@ -237,27 +240,27 @@ public void cleanup(final String projectPath, final List<String> paths,
...
-
-                    @Override
-                    protected void onFailure(Throwable exception) {
-                        callback.onFailure(exception);
+               false, // @param ignoreExternals whether or not to ignore externals (--ignore-externals)
+               false, // @param showIgnored whether or not to show ignored paths (--no-ignored)
+               false, // @param showUpdates whether or not to show repository updates (--show-updates)
+               false, // @param showUnversioned whether or not to show unversioned paths (--quiet)
+               false, // @param verbose whether or not to be verbose (--verbose)
+               new ArrayList<String>(),
```
Comment (skabashnyuk) [inline plugin-svn/che-plugin-svn-ext-subversion/src/main/java/org/eclipse/che/ide/ext/svn/client/SubversionClientServiceImpl.java]: Collections.emtpyList()?

Comment (vzhukovskii) [inline plugin-svn/che-plugin-svn-ext-subversion/src/main/java/org/eclipse/che/ide/ext/svn/client/SubversionClientServiceImpl.java]: This block of code doesn't related to my issue. Maybe we can use Collections class

File: plugin-svn/che-plugin-svn-ext-subversion/src/test/java/org/eclipse/che/ide/ext/svn/server/SubversionApiITest.java [hunk 2]
```diff
@@ -140,11 +148,46 @@ public void testExport() throws Exception {
...
     }
 
     /**
+     * Tests for {@link SubversionApi#propset(PropertySetRequest)} and {@link SubversionApi#propdel(PropertyDeleteRequest)}.
+     *
+     * @throws Exception
+     *         if anything goes wrong
+     */
+    @Test
+    public void testPropSetAndDel() throws Exception {
```
Comment (skabashnyuk) [inline plugin-svn/che-plugin-svn-ext-subversion/src/test/java/org/eclipse/che/ide/ext/svn/server/SubversionApiITest.java]: This test have to be split on two tests

File: plugin-svn/che-plugin-svn-ext-subversion/src/test/java/org/eclipse/che/ide/ext/svn/client/property/PropertyEditorPresenterTest.java [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+import org.mockito.Mock;
+
+import static org.mockito.Mockito.verify;
+
+/**
+ * Unit tests for {@link org.eclipse.che.ide.ext.svn.client.property.PropertyEditorPresenter}.
+ *
+ * @author Vladyslav Zhukovskyi
+ */
+public class PropertyEditorPresenterTest extends BaseSubversionPresenterTest {
```
Comment (skabashnyuk) [inline plugin-svn/che-plugin-svn-ext-subversion/src/test/java/org/eclipse/che/ide/ext/svn/client/property/PropertyEditorPresenterTest.java]: Any test for edit or delete functionality?

**Comment by skabashnyuk** [comment]
PropertyDeleteRequest and PropertyEditRequest do thay have common parts?

File: plugin-svn/che-plugin-svn-ext-subversion/src/test/java/org/eclipse/che/ide/ext/svn/server/SubversionApiITest.java (hunk 2 above)
Comment (vzhukovskii) [inline plugin-svn/che-plugin-svn-ext-subversion/src/test/java/org/eclipse/che/ide/ext/svn/server/SubversionApiITest.java]: Agree, but for deletion property we should set it at first.

File: plugin-svn/che-plugin-svn-ext-subversion/src/test/java/org/eclipse/che/ide/ext/svn/client/property/PropertyEditorPresenterTest.java (hunk 3 above)
Comment (vzhukovskii) [inline plugin-svn/che-plugin-svn-ext-subversion/src/test/java/org/eclipse/che/ide/ext/svn/client/property/PropertyEditorPresenterTest.java]: On client side? We can show form with two radio button which response for deletion and setting specified property and its all.

**Comment by vzhukovskii** [comment]
Ideologically yes, property delete request doesn't contain property value part

File: plugin-svn/che-plugin-svn-ext-subversion/src/test/java/org/eclipse/che/ide/ext/svn/server/SubversionApiITest.java (hunk 2 above)
Comment (skabashnyuk) [inline plugin-svn/che-plugin-svn-ext-subversion/src/test/java/org/eclipse/che/ide/ext/svn/server/SubversionApiITest.java]: it have to be independent, no?

File: plugin-svn/che-plugin-svn-ext-subversion/src/main/java/org/eclipse/che/ide/ext/svn/client/SubversionClientServiceImpl.java (hunk 1 above)
Comment (skabashnyuk) [inline plugin-svn/che-plugin-svn-ext-subversion/src/main/java/org/eclipse/che/ide/ext/svn/client/SubversionClientServiceImpl.java]: and?

**Comment by skabashnyuk** [comment]
Can you combine or reuse something?

**Pull request closed without merging**